### PR TITLE
remove unnecessary variable attribution and dead code

### PR DESCRIPTION
**Pull request by changchengx**

### CODE REVIEW

**Review by yosefe — DISMISSED** [review]

File: src/uct/ib/ud/base/ud_ep.c [hunk 1]
```diff
@@ -671,12 +671,12 @@ uct_ud_ep_connect_to_ep_v2(uct_ep_h tl_ep,
...
-                                                       uct_ud_iface_t);
-    const uct_ib_address_t *ib_addr   = (const uct_ib_address_t*)dev_addr;
-    const uct_ud_ep_addr_t *ep_addr   = (const uct_ud_ep_addr_t*)uct_ep_addr;
-    uct_ib_device_t UCS_V_UNUSED *dev = uct_ib_iface_device(&iface->super);
+    uct_ud_ep_t *ep                  = ucs_derived_of(tl_ep, uct_ud_ep_t);
+    uct_ud_iface_t *iface            = ucs_derived_of(ep->super.super.iface,
+                                                      uct_ud_iface_t);
+    const uct_ib_address_t *ib_addr  = (const uct_ib_address_t*)dev_addr;
+    const uct_ud_ep_addr_t *ep_addr  = (const uct_ud_ep_addr_t*)uct_ep_addr;
+    uct_ib_device_t *dev             = uct_ib_iface_device(&iface->super);
```
Comment (brminich) [inline src/uct/ib/ud/base/ud_ep.c:679]: indentation contains extra space in each row now

File: src/uct/sm/scopy/cma/cma_ep.c [hunk 2]
```diff
@@ -89,10 +89,9 @@ ucs_status_t uct_cma_ep_tx(uct_ep_h tl_ep, const uct_iov_t *iov, size_t iov_cnt,
                            uint64_t remote_addr, uct_rkey_t rkey,
                            uct_scopy_tx_op_t tx_op)
 {
-    uct_cma_ep_t *ep                   = ucs_derived_of(tl_ep, uct_cma_ep_t);
-    size_t local_iov_idx               = 0;
-    size_t UCS_V_UNUSED remote_iov_idx = 0;
-    size_t local_iov_cnt               = UCT_SM_MAX_IOV;
+    uct_cma_ep_t *ep         = ucs_derived_of(tl_ep, uct_cma_ep_t);
+    size_t local_iov_idx     = 0;
+    size_t local_iov_cnt     = UCT_SM_MAX_IOV;
```
Comment (brminich) [inline src/uct/sm/scopy/cma/cma_ep.c:94]: extra spaces

Comment (changchengx) [reply src/uct/sm/scopy/cma/cma_ep.c]: Done

File: src/uct/ib/ud/base/ud_ep.c (hunk 1 above)
Comment (changchengx) [reply src/uct/ib/ud/base/ud_ep.c]: Done

**Comment by changchengx** [comment]
@yosefe @brminich  I've changed this RP according to Mikhail's comments. Because all the change is small, I did force push here.

**Review by yosefe — APPROVED** [review]

**Pull request merged**